packages/gatsby-plugin-image/package.json

```diff
@@ -71,9 +71,10 @@
     "react-dom": "^16.9.0 || ^17.0.0"
   },
   "dependencies": {
+    "@babel/code-frame": "^7.12.13",
```

Comment:
Just want to double check that this isn't re-exported from core. I expect not, but worth asking.

Reply:
Doesn't appear to be